src/action/action.ts

```diff
@@ -124,12 +124,9 @@ export class Action {
   }
 
   async downloadDiskImage(): Promise<string> {
-    core.info(
-      `Downloading disk image: ${this.operatingSystem.virtualMachineImageUrl}`
-    )
-    const result = await cache.downloadTool(
-      this.operatingSystem.virtualMachineImageUrl
-    )
+    const imageURL = `this.input.imageURL !== '' ? this.input.imageURL : this.operatingSystem.virtualMachineImageUrl`
```

Comment:
This doesn't look right. The whole expression on the right hand side is a string.

Reply:
oops, yes you are right. 
Fixed.